src/Miunie.Discord/Embeds/EmbedConstructor.cs

```diff
@@ -52,7 +52,7 @@ public static Embed ToEmbed(this MiunieUser mUser, ILanguageProvider lang)
 
             return new EmbedBuilder()
                 .WithColor(new Color(236, 64, 122))
-                .WithTitle(lang.GetPhrase(PhraseKey.USER_EMBED_TITLE.ToString()))
+                .WithTitle($"**{mUser.Name.ToUpper()}'S** {lang.GetPhrase(PhraseKey.USER_EMBED_TITLE.ToString())}")
```

Comment:
So actually... We kinda want to do this in a different way.
You should add the name as a `{0}` parameter to the Strings resource and then pass the username as an argument.

Also `'s` will not always be the case. For example `Spelos` should be `Spelos'` not `Spelos's`.
You'd want to pass the username in a possessive form into the GetPhrase since I don't think you can format it in the string itself.

Should you have any questions, feel free to @ me. :blush:

Reply:
@petrspelos Can i create anywhere some converter to possessive form?

Reply:
Yes, you can even have it as a private method in this class.

Reply:
@petrspelos Updated